README.md

```diff
@@ -32,11 +32,15 @@ This is a basic everyday Calculator with following features:
 
 
 # Run Locally
+- From the 'Code' section of your forked repo, copy the URL of your forked repo under HTTPS/ SSH/ Github CLI.
+- Open Git Bash terminal.
+- Change the current working directory to the location where you want the cloned directory.
+- Type ```git clone```, and then paste the URL you copied earlier. It will look like this, with your GitHub username instead of ```YOUR-USERNAME```:
 
-- Clone your forked repository
   ```
   git clone https://github.com/YOUR-USERNAME/CALCULATOR-WEBSITE
   ```
+- Press Enter. You local clone is created now.
```

Comment:
No need of this @Arcturus22 so remove that also

Reply:
Alright